features/merge_articles.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Manage Categories
+  As a blog administrator
+  In order to condense similar articles
+  I want to be able to merge articles
+
+  Background:
+    Given the blog is set up
+    And I am logged into the admin panel
+
+  Scenario: Creating a new article
+    Given I am on the new article page
```

Comment:
Can you use something like you created for "I have categories" here instead of going through all of the steps to create a new article? Since you're testing merging not the creation (i think)